pkgs/development/tools/backblaze-b2/default.nix

```diff
@@ -1,31 +1,34 @@
-{ stdenv, lib, fetchFromGitHub, pkgs }:
+{ fetchFromGitHub, pythonPackages, stdenv }:
 
-stdenv.mkDerivation rec {
-  name = "backblaze-b2-0.3.10";
+pythonPackages.buildPythonPackage rec {
+  name = "backblaze-b2-${version}";
+  version = "0.4.4";
 
   src = fetchFromGitHub {
     owner = "Backblaze";
     repo = "B2_Command_Line_Tool";
-    rev = "b097f0f04d3f88d7a372b50ee6db1f89a5249028";
-    sha256 = "1rcy8180476cpmrbls4424qbq8nyq7mxkfikd52a8skz7rd5ljc6";
+    rev = "74a5e567925899f1fc6204aa85d4c84c0d0e511a";
+    sha256 = "1g9j5s69w6n70nb18rvx3gm9f4gi1vis23ib8rn2v1khv6z2acqp";
   };
-  
-  buildInputs = with pkgs; [ python2 ];
 
-  doCheck = true;
+  propagatedBuildInputs = with pythonPackages; [ six ];
+
   checkPhase = ''
     python test_b2_command_line.py test
   '';
 
-  installPhase = ''
-    install -Dm755 b2 "$out/bin/backblaze-b2"
+  postInstall = ''
+    mv "$out/bin/b2" "$out/bin/backblaze-b2"
+
+    mkdir -p "$out/etc/bash_completion.d"
+    cp contrib/bash_completion/b2 "$out/etc/bash_completion.d/"
```

Comment:
Why not just use install rather then `mkdir &&cp`? `install -Dm644 contrib/bash_completion/b2 "$out/etc/bash_completion.d/backblaze-b2"` should do it. Although I guess for nix using `install` isn't that important.